src/bowling.js

```diff
@@ -0,0 +1,49 @@
+class Game {
+
+  constructor(){
+    this.rolls = [];
+  }
+
+  roll(pins) {
+    this.rolls.push(pins)
+  }
+
+  get score() {
+    let score = 0;
+    let rollIndex = 0;
+
+    for (let frameIndex = 0; frameIndex < 10; frameIndex++) {
+      
+      if (this.isStrike(rollIndex)) {
+        score += this.strikeBonus(rollIndex);
+        rollIndex++;
+        continue;
```

Comment:
Haven't seen the 'continue;' keyword before. Is there a reason why its needed here but not in other places?